doc/format.rst

```diff
@@ -690,11 +697,11 @@ belong in docstrings.
 Other points to keep in mind
 ----------------------------
 * Equations : as discussed in the :ref:`Notes <notes>` section above, LaTeX
-  formatting should be kept to a minimum.  Often it's possible to show equations as
-  Python code or pseudo-code instead, which is much more readable in a
-  terminal.  For inline display use double backticks (like ``y = np.sin(x)``).
-  For display with blank lines above and below, use a double colon and indent
-  the code, like::
+  formatting should be kept to a minimum.  Often it's possible to show
+  equations as Python code or pseudo-code instead, which is much more readable
+  in a terminal.  For inline display of code use double backticks,
+  like ````y = np.sin(x)````. For display with blank lines above and below,
```

Comment:
I think you need a space between each pair of double backticks, or maybe backslashes, that does not render properly on github rendered view for me.

Reply:
Nevermind its an issue with the github renderer.